### PR TITLE
[do not merge yet] Connect to Kami

### PR DESCRIPTION
**Pull request by samuelgruetter**
I made sure I can import Kami files in the file `processor/src/Test.v`, now I want to see if this also works on CI.

### CODE REVIEW

**Comment by joonwonc** [comment]
Now Kami compiles with Coq 8.9.0 from the commit point below:
https://github.com/mit-plv/kami/commit/f80b22b179670b6f904734eb48d89d4b9e60f85d

**Comment by samuelgruetter** [comment]
Cool! Feel free to update this `connect-to-kami-2` branch with the new Kami version, and with other changes you might make

File: Makefile [hunk 1]
```diff
@@ -28,6 +28,12 @@ coqutil:
 clean_coqutil:
 	$(MAKE) -C $(DEPS_DIR)/coqutil clean
 
+kami:
+	$(MAKE) -C $(DEPS_DIR)/kami
+
+clean_kami:
+	$(MAKE) -C $(DEPS_DIR)/kami clean
+
```
Comment (samuelgruetter) [inline Makefile]: Detail: clean_kami should be called by clean too

Comment (joonwonc) [reply Makefile]: Ah I forgot that; just pushed the change, thanks!

**Pull request merged**